modules/digitalocean/doks/cluster.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kubernetes.digitalocean.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: digitalocean-doks
+spec:
+  forProvider:
+    name: digitalocean-providerconfig
+    nodePool:
+      - name: worker-pool
+        nodeCount: 3
+        size: s-2vcpu-2gb
+        taint:
+          - effect: NoSchedule
+            key: workloadKind
+            value: database
+    region: nyc1
+    version: 1.22.8-do.1
```

Comment:
I see in the upstream, the latest version is [1.33.1-do.0](https://docs.digitalocean.com/products/kubernetes/details/changelog/#1331-do0-june-16-2025) hope the latest provider config offers it as well, if so we can adjust the version here to latest